src/gettingstarted/own-code/connect-services.md

```diff
@@ -0,0 +1,157 @@
+# Import your own code
+
+## Connect to Services
+
+> **Note:** If your application does not require access to services and you left `.platform/services.yaml` blank, feel free to proceed to the next step.
+
+At this point you have configured your application's services as well as its build and deploy process in your `.platform/services.yaml` and `.platform.app.yaml` files. As an example, in your `.platform.app.yaml` file you may have defined a relationship called `database`:
+
+```yaml
+relationships:
+    database: "mysqldb:mysql"
+```
```

Comment:
Relationships were never mentioned on any of the previous pages.  They should be mentioned/explained either here or on the services.yaml page or the app.yaml page.  I'm not sure which.